core/app/modules/import-export/assets/js/pages/export/export-plugins/components/export-plugins-selection/export-plugins-selection.js

```diff
@@ -0,0 +1,43 @@
+import { useContext, useEffect } from 'react';
+
+import { ExportContext } from '../../../../../context/export-context/export-context-provider';
+
+import PluginsSelection from '../../../../../shared/plugins-selection/plugins-selection';
+import Loader from '../../../../../ui/loader/loader';
+
+import usePlugins, { PLUGIN_STATUS_MAP } from '../../../../../hooks/use-plugins';
+import usePluginsData, { PLUGINS_KEYS } from '../../../../../hooks/use-plugins-data';
+
+export default function ExportPluginsSelection() {
+	const exportContext = useContext( ExportContext ),
+		{ response } = usePlugins(),
+		{ pluginsData } = usePluginsData( response.data ),
```

Comment:
Check for re-render (if need to add useMemo).

Reply:
Changed the structure of the components + prevented the entire component re-render.